app/views/layouts/application.html.erb

```diff
@@ -130,7 +130,7 @@
         <%= content_for?(:content) ? yield(:content) : yield %>
       </main>
     </div>
-
+    <%= render partial: "layouts/feedback" %>
     <%= render partial: "layouts/footer" %>
```

Comment:
the designs (https://www.figma.com/file/zeKXCOjR1DuXXIXpWHaDwS/CORE-Dev-Handover?node-id=1595%3A39150&t=3o7bIOsRN7toymMV-0) seem to remove the source code and cookies links from the footer but that wasn't mentioned in the ticket so I assume that's just a mistake and have not made that change